lib/nuget-parser/index.ts

```diff
@@ -58,7 +71,11 @@ export async function buildDepTreeFromFiles(
   const tree = {
     dependencies: {},
     meta: {},
-    name: path.basename(root || projectRootFolder),
+    name: getRootName(
```

Comment:
Would `projectName` make sense over `rootName`?

Reply:
I think root name is better, it's the root of tree here. Hasn't become a Snyk 'project' yet.